bin/test-all-examples

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+if [ $# -eq 0 ]; then
+  echo "usage: $0 /path/to/exercise"
```

Comment:
going to make indentation consistent, at 4 spaces.

I don't care whether we choose 4 or 2 as long as we are consistent.